src/index.d.ts

```diff
@@ -1317,12 +1317,18 @@ declare namespace admin.machineLearning {
     /**
      * Wait for the model to be unlocked.
      *
-     * @param {number} maxTimeSeconds The maximum time in seconds to wait.
+     * @param {number} maxTimeMillis The maximum time in milliseconds to wait.
+     *     If not specified, a default maximum of 2 minutes will be used.
```

Comment:
We try to avoid "will."  Suggest just "is used."

Reply:
done